src/resolvers/user_management.js

```diff
@@ -25,27 +25,54 @@ export default {
 
       return true
     },
-    login: async (parent, { email, password }, { driver, req, user }) => {
+    login: async (_, { email, password }, { driver, req, user }) => {
```

Comment:
What renaming the parent?

Reply:
we are not using it, so I thought I could use  a convention that I have seen, where the _ indicates that I'm not going to use that parameter

Reply:
it also helps with the linter